gdbinit.py

```diff
@@ -7,11 +7,13 @@
 import site
 import sys
 import time
+import warnings
 from glob import glob
 from os import environ
 from os import path
 
 _profiler = cProfile.Profile()
+warnings.simplefilter("default", UnicodeWarning)
```

Comment:
Is line this necessary? I've never used this library, but the docs say this `UnicodeWarning` should already have the default action.